docs/DeveloperGuide.md

```diff
@@ -27,6 +27,13 @@ a Command Line Interface (CLI).
 
 ### 1.4 Typical User Workflow
 
+![Activity Diagram of DonnaFin](diagrams/ActivityDiagram.png)
+
+This Activity Diagram shows how one might navigate through the DonnaFin application in a typical usage.
+
+
+Commands refer to the pre-defined functions that are used by the user. Invalid commands refer to commands that do not
+exist or commands that are used with the wrong format.
```

Comment:
Invalid commands refer to commands that are not available in the current window or commands that are used with the wrong format.